### ISSUE TITLE
Out prediction is blank

### ISSUE THREAD

**Issue by ychnh** [holmdk/Video-Prediction-using-PyTorch#3]
When looking at the prediction of evaluation the outputs of the model are just blank.
Were you able to train the model to predict expected frames?
The loss goes down but the model seems to have converged to optimally predicting blanks.

**Comment by holmdk** [comment]
Yes, I have experienced this as well. This is primarily a problem for MovingMNIST, since the data is very sparse. Have you tried altering the learning rate? It sometimes happens if I set it too low, or if the weight initialization causes a local minima. You could also change the output to be 0's and 1's, and then change the loss function to CrossEntropy and set a higher weight on the positive class. For this you would also need to change the output classes from 1 (regression) to 2 (binary classification).
